includes/bp-media-function.php

```diff
@@ -908,26 +905,62 @@ function bp_media_pagination_links( $query ) {
 	echo bp_media_get_pagination_links( $query );
 }
 
-	function bp_media_get_pagination_links( $query ) {
+function bp_media_get_pagination_links( $query ) {
 
-			$paged = ( isset($_GET['mpage']) ) ? $_GET['mpage'] : 1;
+	$paged = ( isset( $_GET['mpage'] ) ) ? $_GET['mpage'] : 1;
 
-			$pag_args = array(
-				'mpage' => '%#%'
-			);
+	$pag_args = array(
+		'mpage' => '%#%',
+	);
 
-			if ( defined( 'DOING_AJAX' ) && true === (bool) DOING_AJAX ) {
-				$base = remove_query_arg( 's', wp_get_referer() );
-			} else {
-				$base = '';
-			}
+	if ( defined( 'DOING_AJAX' ) && true === (bool) DOING_AJAX ) {
+		$base = remove_query_arg( 's', wp_get_referer() );
+	} else {
+		$base = '';
+	}
```

Comment:
I'll go ahead and do PHPCS clean on the entire plugin. Will take me a bit but should solve some issues and do some validation updates.